src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int NUMBER_IN_THE_ARRAY = 1;
+    private static final int ARRAY_IN_THE_ARRAY = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String data = readFile(fromFileName);
+        String report = createReport(data);
+        writeToFile(toFileName, report);
+    }
+
+    private String readFile(String fromFileName) {
+        StringBuilder fileData = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String str = bufferedReader.readLine();
+            while (str != null) {
+                fileData.append(str).append(System.lineSeparator());
+                str = bufferedReader.readLine();
+            }
```

Comment:
we can prettify it a bit  :)
![image](https://github.com/mate-academy/jv-work-with-file/assets/49767049/b53517e4-50f7-4549-8332-61a9b0dd1997)